README.md

```diff
@@ -144,6 +146,8 @@ show_decimal: false
 | use_browser_time      | boolean          | **Optional** | Uses the time from your browser to indicate the current time. If not provided, uses the [time_zone](https://www.home-assistant.io/blog/2015/05/09/utc-time-zone-awareness/#setting-up-your-time-zone) configured in HA            | `false`   |
 | time_zone             | string           | **Optional** | Uses the given [time zone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) to indicate the current date and time. If not provided, uses the time zone configured in HA                                              | `null`    |
 | show_decimal       | boolean          | **Optional** | Displays main temperature without rounding                                                                                                                                                                                     | `false`   |
+| apparent_sensor    | string           | **Optional** | ID of the apparent temperature sensor entity. Used to show the apparent temperature based on a sensor value                                                                                                 | `''`      |
+| show_apparent       | boolean          | **Optional** | Displays the apparent (real feel, feels like) temperature                                                                                                                                                                                     | `false`   |
```

Comment:
wouldn't it be easier for the user to only have one property to configured?
one can safely assume that the users wants to show the apparent temperature if they configure a sensor for it.
what do you think?

Reply:
i was thinking about it when making this, i kinda hacked it together for myself so will polish it up now.